modules/prebidServerBidAdapter/index.js

```diff
@@ -561,8 +555,8 @@ Object.assign(ORTB2.prototype, {
       this.adUnitsByImp[impressionId] = adUnit;
 
       const nativeParams = adUnit.nativeParams;
-      let nativeAssets;
-      if (nativeParams) {
+      let nativeAssets = nativeAssetCache[impressionId] = deepAccess(nativeParams, 'ortb.assets');
```

Comment:
GH does not let me pick the right line for this - further down here, line 681, the code path for native packages only `ortb.assets` into an ORTB native request. Does that still make sense, or should it just say `request.native = nativeParams.ortb`? if no, why not? maybe explained in the form of a unit test? :)

Reply:
You are right! Before, there was no way for publishers to pass context, plcmttype and other fields into a media type object. With my latest commit the mediaType.native.ortb will contain exactly that, so if that's present, i'm going to pass that to the request, otherwise we go the old way.